kuwala/core/canvas/README.md

```diff
@@ -0,0 +1,11 @@
+# Kuwala Canvas [EXPERIMENTAL]
+
+Kuwala Canvas aims to be an intuitive tools to create a data analysis starting from collecting data
+from data sources until visualizing it in graphs.
+
+To Run the Project:
+
+1. Make sure that you have node installed on your machine
+2. npm install
```

Comment:
Should be `npm ci` to install the correct versions specified in the `package-lock.json`.